transports/sql/index.md

```diff
@@ -134,3 +135,10 @@ When the SQL Server transport is used in combination with [NHibernate persistenc
 SQL Server transport supports all [transaction handling modes](/transports/transactions.md), i.e. Transaction scope, receive only, send atomic with receive, and no transactions.
 
 Refer to [transport transactions](/transports/transactions.md) for a detailed explanation of the supported transaction handling modes and available configuration options.
+
+## Time-to-be-received
+
+The SQL transport runs a periodic task that removes expired messages from the queue. The task is first executed when the endpoint starts and is subsequently scheduled to execute 5 minutes after the previous run when the task has been completed. Expired messages are not received from the queue and their disk space will be reclaimed when the periodic task executes.
```

Comment:
Not any more. It has been removed because it tended to deadloack with the receive tasks.